From 91e32211548fbf11a118c8961c609486ca0e6f31 Mon Sep 17 00:00:00 2001
From: Martin Erik Werner <martinerikwerner@gmail.com>
Date: Tue, 31 Mar 2015 12:11:53 +0200
Subject: [PATCH] system-install: Fix binary name

Since the main makefile now produces binaries with suffixes in src/,
make sure the system-install targets use these filenames.
---
 src/system-install.mk | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/system-install.mk b/src/system-install.mk
index 568df6a..e6eae10 100644
--- a/src/system-install.mk
+++ b/src/system-install.mk
@@ -2,8 +2,8 @@ appname=$(APPNAME)
 appnamefull=$(shell sed -n 's/.define VERSION_NAME *"\([^"]*\)"/\1/p' version.h)
 appsrcname=$(APPNAME)
 cappname=$(shell echo $(appname) | tr '[:lower:]' '[:upper:]')# Captial appname
-appclient=$(APPCLIENT)
-appserver=$(APPSERVER)
+appclient=$(APPCLIENT)$(APPMODIFIER)$(BIN_SUFFIX)
+appserver=$(APPSERVER)$(APPMODIFIER)$(BIN_SUFFIX)
 
 prefix=/usr/local
 games=
-- 
2.1.4

